draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -123,38 +133,37 @@ Entry:
 
 Receipt:
 
-: A COSE Object containing the header parameters necessary to convey a proof types for verifiable data structures.
+: A COSE object, as defined in {{RFC9052}}, containing the header parameters necessary to convey VDP for an associated VDS.
 
 
 # Verifiable Data Structures in CBOR {#sec-generic-verifiable-data-structures}
 
 This section describes representations of verifiable data structure proofs in CBOR.
 For example, construction of a merkle tree leaf, or an inclusion proof from a leaf to a merkle root, might have several different representations, depending on the verifiable data structure used.
 Differences in representations are necessary to support efficient verification, unique security or privacy properties, and for compatibility with specific implementations.
-In order to improve interoperability we define two extension points for enabling verifiable data structures with COSE, and we provide concrete examples for the structures and proofs defined in {{-certificate-transparency-v2}}.
+This document defines two extension points for enabling verifiable data structures with COSE and provides concrete examples for the structures and proofs defined in {{-certificate-transparency-v2}}.
 The design of these structures is influenced by the conventions established for COSE Keys.
 
+During testing and development the experimental range SHOULD be used, unless early assignment for a provisional entry has been completed.
+
 ## Structures {#sec-cose-verifiable-data-structures}
 
 Similar to [COSE Key Types](https://www.iana.org/assignments/cose/cose.xhtml#key-type), different verifiable data structures support different algorithms.
 As EC2 keys (1: 2) support both digital signature and key agreement algorithms, RFC9162_SHA256 (TBD_1 : 1) supports both inclusion and consistency proofs.
 
 This document establishes a registry of verifiable data structure algorithms, with the following initial contents:
 
-* Name: The name of the verifiable data structure
-* Value: The identifier for the verifiable data structure
-* Description: The identifier for the verifiable data structure
-* Reference: Where the verifiable data structure is defined
-
 | Name            | Value | Description                      | Reference
 |---
 | N/A             | 0     | N/A                              | N/A
 | RFC9162_SHA256  | 1     | SHA256 Binary Merkle Tree        | {{-certificate-transparency-v2}}
```

Comment:
Whats the deal with the experimental slots ?

Reply:
https://github.com/cose-wg/draft-ietf-cose-merkle-tree-proofs/pull/18/files#diff-e456ca8fff23262339949e1ef0e3b1f5c757b63b05b691c26d656654bef76d6fR148

^ if this is not clear, please suggest changes to it